packages/compat/test/index.test.ts

```diff
@@ -23,7 +23,7 @@ import { NodeTracerProvider } from "@opentelemetry/sdk-trace-node"
 import { SemanticAttributes } from "@opentelemetry/semantic-conventions"
 import { beforeEach, describe, expect, it } from "@solarwinds-apm/test"
 
-import { instrument, pInstrument } from "../src"
```

Comment:
ESM requires full file name and extensions in import. We use `.js` and not `.ts` cause TypeScript (by design) never touches file extensions in imports